modules/installation-osp-adding-bare-metal.adoc

```diff
@@ -0,0 +1,137 @@
+
+* Bare metal is available as link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/bare_metal_provisioning/sect-configure#creating_the_bare_metal_flavor[an {rh-openstack} flavor].
+
+* You deployed a {product-title} cluster on installer-provisioned infrastructure.
+
+* Your {rh-openstack} cloud provider is configured to route traffic between the installer-created VM
+subnet and the pre-existing bare metal subnet.
+
+.Procedure
+
+. Create a file called `baremetalMachineSet.yaml`, and then add the bare metal flavor to it:
++
+.A sample bare metal MachineSet file
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructure_ID> 
+    machine.openshift.io/cluster-api-machine-role: <node_role> 
+    machine.openshift.io/cluster-api-machine-type: <node_role> 
+  name: <infrastructure_ID>-<node_role> 
+  namespace: openshift-machine-api
+spec:
+  replicas: <number_of_replicas>
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <infrastructure_ID> 
+      machine.openshift.io/cluster-api-machineset: <infrastructure_ID>-<node_role> 
+  template:
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructure_ID> 
+        machine.openshift.io/cluster-api-machine-role: <node_role> 
+        machine.openshift.io/cluster-api-machine-type: <node_role> 
+        machine.openshift.io/cluster-api-machineset: <infrastructure_ID>-<node_role> 
+    spec:
+      providerSpec:
+        value:
+          apiVersion: openstackproviderconfig.openshift.io/v1alpha1
+          cloudName: openstack
+          cloudsSecret:
+            name: openstack-cloud-credentials
+            namespace: openshift-machine-api
+          flavor: <nova_flavor>
+          image: <glance_image_name_or_location>
+          kind: OpenstackProviderSpec
+          networks:
+          - filter: {}
+            subnets:
+            - filter:
+                name: <subnet_name>
+                tags: openshiftClusterID=<infrastructure_ID>
+          securityGroups:
+          - filter: {}
+            name: <infrastructure_ID>-<node_role>
+          serverMetadata:
+            Name: <infrastructure_ID>-<node_role>
+            openshiftClusterID: <infrastructure_ID>
+          tags:
+          - openshiftClusterID=<infrastructure_ID>
+          trunk: true
+          userDataSecret:
+            name: <node_role>-user-data 
+----
+
+. On a command line, to create the MachineSet resource, type:
++
+[source,terminal]
+----
+oc create -v baremetalMachineSet.yaml
+----
+
+You can now use bare-metal compute machines in your {product-title} cluster.
+
+// So... is this first task necessary?
+// - Create install-config.yaml:
```

Comment:
Does keeping this from the upstream docs make sense? Isn't this a post-installation task?

Reply:
Follow-up: added this module to the post-installation tasks assembly.